e2e/specs/notification_trigger/helpers.js

```diff
@@ -0,0 +1,28 @@
+const {asyncSleep} = require('../../modules/utils');
+
+export async function triggerTestNotification(firstServer) {
```

Comment:
Just curious - is there a reason we aren't just testing this in the web app? Is there a major difference in behaviour between that and the Desktop App?

Reply:
There isn't any difference. I wanted to test if the app's icon shows unread badge. I could only do it for macOS, but not for others. 

I kept just 1 positive test case to be checked on all OS versions.
I can remove if you feel its wont add much value, since similar flow in in web app

Reply:
Ahh okay, no that's a good test to add, even if just to be thorough.